content/blogposts/2023/announcing-scip-clang.md

```diff
@@ -0,0 +1,84 @@
+# Announcing scip-clang: a new SCIP indexer for C and C++
+
+[scip-clang](https://github.com/sourcegraph/scip-clang) is our new indexer
```

Comment:
FWIW, I don't really like reading posts written in this style, so if there's some specific bits you'd like me to adopt, please point them out. Otherwise, to me it's just a bit too punchy rather than being grounded.

Reply:
Nothing in particular, I just wanted to see it's output.  Rereading with your changes feels good.